prac_07/myguitars.py

```diff
@@ -0,0 +1,47 @@
+from guitar import Guitar
+
+
+def main():
+    file_name = 'guitars.csv'
+    guitars = load_guitars(file_name)
+
+    print("All Guitars:")
+    for guitar in guitars:
+        print(guitar)
+
+    guitars.sort()
+
+    print("\nSorted Guitars (Oldest to Newest):")
+    for guitar in guitars:
+        print(guitar)
+
+    new_guitar_name = input("Enter the name of your new guitar: ")
+    new_guitar_year = int(input("Enter the year of your new guitar: "))
+    new_guitar_cost = float(input("Enter the cost of your new guitar: "))
+    new_guitar = Guitar(new_guitar_name, new_guitar_year, new_guitar_cost)
+    guitars.append(new_guitar)
+
+    save_guitars(file_name, guitars)
+    print("\nNew guitar added and saved to file.")
+
+
+def load_guitars(file_name):
```

Comment:
It defines the load_guitars() function, which reads guitar data from the CSV file and returns a list of Guitar instances.